source/_patterns/01-molecules/components/about-profile.mustache

```diff
@@ -16,7 +16,7 @@
       {{/fallback}}
     {{/image}}
     {{^image}}
-      <img src="{{#assetRewrite}}{{assetsPath}}/img/patterns/molecules/about-page-profile-placeholder.svg{{/assetRewrite}}" alt="" class="about-profile__image">
+      <img src="{{#assetRewrite}}{{assetsPath}}/img/patterns/molecules/image-placeholder.svg{{/assetRewrite}}" alt="" class="about-profile__image">
```

Comment:
I think we should call it `profile-image-placeholder.svg` otherwise it's not clear from the image name what it is a placeholder for.

Reply:
Done